src/compiler/utilitiesPublic.ts

```diff
@@ -1379,6 +1379,10 @@ namespace ts {
         return node.kind === SyntaxKind.ImportDeclaration;
     }
 
+    export function isDynamicImport(node: Node): node is CallExpression {

```

Comment:
Is it ok to rename interface `ImportCall` to `DynamicImport`?

Reply:
If you do, you might want to leave `ImportCall` in as a type alias to `DynamicImport` with a `/** @deprecated */` comment, since it is part of the public API. I would also call it `DynamicImportExpression` vs. just `DynamicImport`. The naming scheme for various things in the compiler is starting to become a bit arbitrary, so I'd prefer we aim for consistency with `CallExpression` et al.